spark/src/main/scala/geotrellis/spark/ContextRDDMethods.scala

```diff
@@ -13,7 +13,7 @@ class ContextRDDMethods[K: ClassTag, V: ClassTag, M](val rdd: RDD[(K, V)] with M
   def mapKeys[R: ClassTag](f: K => R): RDD[(R, V)] with Metadata[M] =
     rdd.withContext { rdd => rdd.map { case (key, tile) => f(key) -> tile } }
 
-  def mapValues(f: V => V): RDD[(K, V)] with Metadata[M] =
+  def mapValues[W](f: V => W): RDD[(K, W)] with Metadata[M] =
```

Comment:
Curious on how something like this plays out for, say, changing the cell type of the tile. So we would have a cell type in the raster metadata would then be invalid, and it's carried through by default. I'm wondering if we shouldn't override mapValues? Is that going against stuff we decided on in the many long talks we've had about this?